INSTALL.md

```diff
@@ -46,6 +46,24 @@ Additional Qt plugins can be built to extend Meshroom UI features. They can be f
 though they might get better integration in the future.
 Note that they are optional but highly recommended.
 
+For Windows, there may be problems setting `QML2_IMPORT_PATH` to contain multiple paths. The easiest approach may be to duplicate what is done for the binary release. This has a directory structure like this:
```

Comment:
I would like to avoid "may be" in the wording of the documentation. Documentation is there to provide information that people can trust.

From Qt documentation QML2_IMPORT_PATH is supposed to accept multiple paths.
https://doc.qt.io/qt-5/qtqml-syntax-imports.html
Multiple path entries must be separated by ':' under Unix and ';' under Windows. 

Could you double check? If it doesn't work, it would be good to report an issue on the Qt project.

Reply:
I just tried: 

```
set QML2_IMPORT_PATH="D:\Meshroom_Dev\qmlAlembic\qml;D:\Meshroom_Dev\QtOIIO\qml;D:\Meshroom_Dev\QtAliceVision\qml"

python meshroom/ui 
```

and none of the plugins are available.

I've created https://bugreports.qt.io/browse/QTBUG-84390

Reply:
I've reworded the pull request. It's purposely worded more passively as I'm don't regularly build large projects any more, and even less so on Windows. There's a very large chance I've broken something when I used vcpkg built qt :-)

Reply:
And added a link to the QT bug.

Reply:
After follow up on https://bugreports.qt.io/browse/QTBUG-84390, turns out quotes don't work in path specifications.

The things you learn.

Update to remove most of the changes, add a comment re quotes, and tweak the example path specs to add to, rather than replace.